client/api_client.go

```diff
@@ -84,6 +84,9 @@ type ApiClientInterface interface {
 	ModuleDelete(id string) error
 	ModuleUpdate(id string, payload ModuleUpdatePayload) (*Module, error)
 	Modules() ([]Module, error)
+	APIKeyCreate(payload APIKeyCreatePayload) (*APIKey, error)
```

Comment:
Would be best to keep it as pascal case (i.e `ApiKeyCreate`)

Reply:
@yaronya 
arghhh! it looks like ApiKey was already used.
In any case, I renamed lots of stuff. Hopefully it makes more sense now.

https://github.com/env0/terraform-provider-env0/pull/301/commits/29d5364208e8dfde091ebde87dffcbfe75835869

Reply:
4214ea271728fcf7965e3c3e4f10f5baa70ccf9e